### ISSUE TITLE
[Accessibility]: Repo Management - hard to sync public repo using screen reader

### ISSUE THREAD

**Issue by jplahn** [sourcegraph/sourcegraph#36865]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33523

### Problem description

The entire flow of "Syncing specific public repositorities by URL" is very cumbersome and not easy to follow. Part of the challenge is that much of the Settings page is on the same heading level, so you have to cycle all the way to end to finally get to the area of the page where you can manage repositories. 

Once you get into the manage repositories section, there are a lot of items to walk through. The text box to actually type in public repos isn't super clear since it's just called text box which doesn't give you confidence that you're in the right place.

### Expected behavior

I think the whole flow of this issue needs to be improved. It probably starts with better nesting of elements on the Settings page and clear navigating through the manage repositories flow. 

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
As we have recently decided to sunset user-added repositories on [Sourcegraph.com](http://sourcegraph.com/), this audit flow is no longer available. 

Closing this issue as not planned.
